Reviews/513 - On-Chain MVPR/4 - KYC Contract/README.md

```diff
@@ -0,0 +1,205 @@
+
+```sh
+@ggurbet ➜ /workspaces/dao-contracts (7c24f46) $ make build-contracts
+make: *** No rule to make target 'build-contracts'.  Stop.
+```
+
+Upon inspecting the `Makefile`, reviewer found out the correct target name has since been changed to `build-dao-contracts`. This issue was reported to OP and has been updated in the latest versin of the README.
+
+Contacting the OP and after making the necessary updates on the README, reviewer run the build operations again using the `make build-all` command resulted in another error:
+
+```sh
+@ggurbet ➜ /workspaces/dao-contracts (7c24f46 ✗) $ make build-all
+cargo build --release --target wasm32-unknown-unknown --quiet --features=wasm --no-default-features -p casper-dao-contracts
+error[E0463]: can't find crate for `core`
+  |
+  = note: the `wasm32-unknown-unknown` target may not be installed
+  = help: consider downloading the target with `rustup target add wasm32-unknown-unknown`
+  = help: consider building the standard library from source with `cargo build -Zbuild-std`
+
+error: aborting due to previous error
+
+For more information about this error, try `rustc --explain E0463`.
+error: could not compile `rand_core`
+
+To learn more, run the command again with --verbose.
+error[E0463]: can't find crate for `core`
+  |
+  = note: the `wasm32-unknown-unknown` target may not be installed
+  = help: consider downloading the target with `rustup target add wasm32-unknown-unknown`
+  = help: consider building the standard library from source with `cargo build -Zbuild-std`
+
+error: aborting due to previous error
+
+For more information about this error, try `rustc --explain E0463`.
+error: build failed
+make: *** [Makefile:14: build-dao-contracts] Error 101
+```
+
+Running the suggested command:
+
+```sh
+@ggurbet ➜ /workspaces/dao-contracts (7c24f46 ✗) $ rustup target add wasm32-unknown-unknown
+info: downloading component 'rust-std' for 'wasm32-unknown-unknown'
+info: installing component 'rust-std' for 'wasm32-unknown-unknown'
+ 13.4 MiB /  13.4 MiB (100 %)   8.4 MiB/s in  1s ETA:  0s
+```
+
+Then building with `make build-all` successfully built the project:
+
+```sh
+@ggurbet ➜ /workspaces/dao-contracts (7c24f46 ✗) $ make build-all
+cargo build --release --target wasm32-unknown-unknown --quiet --features=wasm --no-default-features -p casper-dao-contracts
+cargo build --release --target wasm32-unknown-unknown --quiet --features=wasm --no-default-features -p casper-dao-erc20
+cargo build --release --target wasm32-unknown-unknown --quiet --features=wasm --no-default-features -p casper-dao-erc721
+```
+
+## Overall Impression of usage testing
+
+Installation using given installation instructions failed to work. After making some small adjustments, reviewer was able to install the contracts. Reviewer suggests updating the README to reflect the installation process properly and also use the default commands used by Casper for consistency in the following milestones.
```

Comment:
Has this been requested of the grantee?

Reply:
This has been updated in the latest README.